csv_to_entities.py

```diff
@@ -0,0 +1,97 @@
+"""
+python csv_to_entites.py
+
+accept in a UTF8 csv file and a delimiter and converts it to entities
+your csv should be of the format
+no headers
+filename without .csv is entity name
+col0 = key_value
+col1 = onward are synonyms
+for example this is the way Dialogflow exports an entity
+
+
+
+"""
+# *********************************************************************************************************************
+
+# standard imports
+import json
+
+# 3rd party imports
+import pandas
+import click
+
+
+@click.command()
+@click.option('-f', '--filename', type=str, required=True,
+              help='Input File Path')
+@click.option('-d', '--delimiter', type=str, required=False, default=",",
+              help='Delimiter for the csv file')
+@click.option('-l', '--language', type=str, required=False, default="en",
+              help='Language of entities default: en')
+def main(filename: str,
+         delimiter: str,
+         language:str
+         ) -> None:
+    """Main Function"""
+
+    # read the input csv with columns 0,1,2 etc
+    assert filename.endswith(".csv")
+    df = pandas.read_csv(filename, encoding='utf8',header=None, delimiter=delimiter)
+
+    # assert
+    assert df.shape[0] >= 1
+    assert df.shape[1] >= 2
+
+    # values
+    values = []
+
+    # iterate through dataframe
+    for i in range(df.shape[0]):
+
+        # work out how many synonyms we have (columns 3 onward until NaN)
```

Comment:
If we have coulm1,col2,col3,...
The synonym should start from column 2  and and the keys values should be taken from col1. Instead the script is taking the key value from col2 and synonyms from col3 onwards.
[simple_entity.csv](https://github.com/zia-ai/academy/files/13317038/simple_entity.csv)
[simple_entity.json](https://github.com/zia-ai/academy/files/13317039/simple_entity.json)